src/librustc_session/config.rs

```diff
@@ -70,6 +70,17 @@ impl FromStr for Sanitizer {
     }
 }
 
+/// Control Flow Guard has three possible states:
+/// "Disabled" => Do not emit Control Flow Guard metadata or checks.
```

Comment:
Nit: You could comment each variant separately.

Reply:
Fixed